repos/system_upgrade/el7toel8/actors/persistentnetnamesconfig/actor.py

```diff
@@ -31,20 +22,6 @@ class PersistentNetNamesConfig(Actor):
     tags = (ApplicationsPhaseTag, IPUWorkflowTag)
     initrd_files = []
 
-    def generate_link_file(self, interface):
```

Comment:
I would leave this here. Or move everything to libs. Decide by yourself, this is not a requirement